scripts/src/main/scala/com/gu/mediaservice/scripts/EsScript.scala

```diff
@@ -13,7 +13,7 @@ object MoveIndex extends EsScript {
   def run(esHost: String, extraArgs: List[String]) {
 
     object EsClient extends ElasticSearchClient {
-      val imagesAlias = "imagesAlias"
+      val imagesAlias = "readAlias"
```

Comment:
I'm not sure tbh. I couldn't get it workings and there was this comment further down in the file: 
  // FIXME: Get from config (no can do as Config is coupled to Play)